numeric-keyboard/src/main/java/com/konaire/numerickeyboard/NumericKeyboard.kt

```diff
@@ -80,6 +80,28 @@ class NumericKeyboard: FrameLayout {
             }
         }
 
+    var keySpecialText: String = ""
+        set(value) {
+            field = value
+            if (childCount > 0) {
+                updateView(getChildAt(0))
+            }
+        }
+
+    var keySpecialListener: OnClickListener? = null
+        set(value) {
+            field = value
+            if (childCount > 0) {
+                updateView(getChildAt(0))
+            }
+        }
+
+    fun onKeySpecialClick(function: (() -> Unit)) {
```

Comment:
This method is overkill. You can just use lambda if you want to add a custom listener, right?